nucypher/cli/characters/ursula.py

```diff
@@ -21,8 +21,8 @@
 from constant_sorrow.constants import NO_BLOCKCHAIN_CONNECTION
 from twisted.internet import stdio
 
-from nucypher.blockchain.eth.interfaces import BlockchainInterface
-from nucypher.blockchain.eth.registry import EthereumContractRegistry
+from nucypher.blockchain.eth.interfaces import BlockchainInterface, BlockchainInterfaceFactory
```

Comment:
So, I suppose, the `BlockchainInterfaceFactory` wasn't used by `ursula.py` at all - hence the problems with blockchain connection